tests/integration-tests/configs/common/common.yaml

```diff
@@ -516,11 +516,11 @@ storage:
         schedulers: ["slurm"]
   test_ebs.py::test_ebs_multiple:
     dimensions:
-      - regions: ["ca-central-1"]
+      - regions: ["eu-west-1"]
```

Comment:
In conftest.py, there's:
# io2 EBS volumes not available in cac1-az4
    "ca-central-1": ["cac1-az1", "cac1-az2"],
Since we remove tests from ca-central-1, should we remove this restriction?
Does all volumes available for all azs in eu-west-1?

Reply:
> Does all volumes available for all azs in eu-west-1?

Good question. I don't know. Do you know how I can find out?

Reply:
> Since we remove tests from ca-central-1, should we remove this restriction?

I'd say not for now, since I think this might be a temporary change until we figure out why we're hitting the errors that we are in ca-central-1.

Reply:
I was trying to create the volumes in the integration test in the azs in eu-west-1, they are able to launch

Reply:
Thank you for checking.